test.sh

```diff
@@ -27,6 +39,8 @@ for dir in ${VERSIONS}; do
   fi
 
   if [ -n "${TEST_CONU_MODE}" ]; then
+    echo "Conu tests are SUPPRESSED now."
+    continue
```

Comment:
We do not want to continue here, the openshift tests would be skipped that way.

Reply:
Maybe we should just remove this block. I do not see us using it again in the future tbh